app/searchers/quick_search/archives_space_searcher.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'uri'
+require 'nokogiri'
+
+module QuickSearch
+  # QuickSearch seacher for UMD Library Website
+  class ArchivesSpaceSearcher < QuickSearch::Searcher
+    def search
+      @response = @http.get uri
+      @results = Nokogiri::HTML response.body
+      @total = total
+    end
+
+    def results
+      return @results_list if @results_list
+
+      @results_list = @results.css(row_selector).map do |row|
+        OpenStruct.new(
+          link: get_hyperlink(row),
+          title: get_title(row),
+          description: get_description(row)
+        )
+      end
+      @results_list
+    end
+
+    def total
+      return '' unless @results.title
+
+      @results.title.match(/(\d+)/).to_s
+    end
+
+    def loaded_link
+      uri.to_s
+    end
+
+    private
+
+    def host
+      QuickSearch::Engine::ARCHIVESSPACE_CONFIG['search_url']
+    end
+
+    def uri
+      base = URI.parse host
+      search_term = Array.wrap(http_request_queries['uri_escaped'] || '')
+      base.query = base_query_params.merge('q' => search_term).to_query
+      base
+    end
+
+    def base_query_params
+      QuickSearch::Engine::ARCHIVESSPACE_CONFIG['query_params']
```

Comment:
For naming consistency, please rename "ARCHIVESSPACE_CONFIG" to "ARCHIVES_SPACE_CONFIG" in this file and in config/initializers/archives_space_searcher_config.rb